README.md

```diff
@@ -31,6 +32,7 @@ give you a working calendar component.
 * `onSelect` — a function that is called whenever user
 * `onSelectionProgress` - a function that is called whenever user changes
 * `selected` — selected dates. Can be ether single `Date` object if `mode` is `single`, or object `{start: Date(), end: Date()}` if `mode` is `range`
+* `selectedDaysLimit` — number of days avaiable for selection from the first selected date (`number`)
```

Comment:
Name is too vague to me. The entity is `range`, so it could be `rangeLimit` or `rangeLengthLimit` or so.